src/water.rs

```diff
@@ -109,7 +109,7 @@ impl WaterTile {
 }
 
 /// Setup water.
-fn setup_water(
+pub fn setup_water(
```

Comment:
Making the systems public would allow me to initialize the systems after loading a certain scenery, and setting the right WaterSettings, which might not be possible to determine while inserting the plugin.